atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueServiceImpl.java

```diff
@@ -1438,6 +1448,10 @@ public void createTables(final Map<TableReference, byte[]> tableNamesToTableMeta
         return filteredTables;
     }
 
+    private void runSchemaMutationTask(Runnable task) {
+        Futures.getUnchecked(schemaMutationExecutor.submit(task));
+    }
```

Comment:
This will change the exception type of what is thrown (from `IllegalStateException` to `UncheckedExecutionException`) - so technically is a break though is probably okay. 

Either way we should document this in release notes / javadoc for `CassandraKeyValueServiceImpl` which has changed.

Reply:
Yep, makes sense (this happens when Cassandra, an AtlasDB dependency, is down).

One last thing - JavaDoc for `createTable` / `createTables` / `dropTable` / `dropTables` in this class should be `@throws AtlasDbDependencyException` rather than `@throws IllegalStateException` after this change right?